client/src/utils/data-formats/formats/scorecard.ts

```diff
@@ -52,15 +52,15 @@ export function scorecard (
 
   let checkValue = (values && values[0] && values[0][countField]) || 0; 
   
-  let createValue = (value: any, heading: string, color: string, icon: string, subvalue?: any, subheading?: string) => {
+  let createValue = (value: any, heading: string, color: string, icon: string, sv?: any, sh?: string) => {
```

Comment:
why use shortcuts? subValue is much better than sv :)

Reply:
new linting errors... will find a better name